src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.service.impl.DataParserImpl;
+import core.basesyntax.service.impl.FileReaderImpl;
+import core.basesyntax.service.impl.FruitOperationStrategy;
+import core.basesyntax.service.impl.ReportServiceImpl;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FileReaderImpl readerService = new FileReaderImpl();
+        DataParserImpl parserService = new DataParserImpl();
+        FruitOperationStrategy strategy = new FruitOperationStrategy();
+        ReportServiceImpl reportService = new ReportServiceImpl();
+
+        parserService.parse(readerService.read("fruits.csv"))
+                .forEach(t -> strategy.getOperationService(t).processOperation(t));
+
+        reportService.createReport();
```

Comment:
Let's have main() method like it described in the requirements
Each method call in new line

![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/6d8ed5fa-ca61-4410-9d55-b003c617c795)